_topics/en/payment-batching.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Payment batching
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Fee Management
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Payment batching** is the technique of putting multiple payments in
+  the same onchain transaction.  This splits the cost of creating a
+  transaction, spending inputs, and creating a change output across all
+  the payments in the transaction, reducing the average cost per payment.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  It's realistically possible to save 75% on transaction fees by
+  batching just a small number of payments and with no degradation in
+  confirmation speed or other changes required.  Even using exactly the
+  same inputs you'd use without batching, it's possible to save more
+  than 20%.
+
+  ![Plot of savings from payment batching](/img/posts/payment-batching/p2wpkh-batching-cases-combined.png)
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: Scaling Bitcoin using Payment Batching
+      link: scaling payment batching
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "2018 year-in-review: about 11% of payments used batching"
+    url: /en/newsletters/2018/12/28/#batched-payments
+
+  - title: "Presentation: A Return to Fees, covering techniques including batching"
+    url: /en/newsletters/2019/05/29/#presentation-a-return-to-fees
+
+  - title: "Proposed new opcode to make batching more efficient during fee spikes"
+    url: /en/newsletters/2019/05/29/#proposed-transaction-output-commitments
+
+  - title: "Non-equal value coinjoins could look like batching for improved privacy"
+    url: /en/newsletters/2020/01/08/#coinjoins-without-equal-value-inputs-or-outputs
+
+  - title: "OP_CHECKTEMPLATEVERIFY discussion about compressed payment batching"
+    url: /en/newsletters/2020/02/12/#op-checktemplateverify-ctv-workshop
+
+  - title: Coinbase withdrawal transactions now using payment batching
```

Comment:
Maybe something like "Withdrawal transactions from Coinbase now use payment batching". Without any context,and when placed at the beginning of the sentence, this could mean Coinbase (exchange) or coinbase (first transaction in a block).